aptos-move/framework/aptos-framework/sources/configs/gas_schedule.spec.move

```diff
@@ -93,10 +93,16 @@ spec aptos_framework::gas_schedule {
     }
 
     spec set_for_next_epoch(aptos_framework: &signer, gas_schedule_blob: vector<u8>) {
+        use aptos_framework::util;
+
+        include system_addresses::AbortsIfNotAptosFramework{ account: aptos_framework };
```

Comment:
This include is not necessary: SetForNextEpochAbortsIf already has the aborts_if condition for the account address.

Reply:
if `config_buffer::SetForNextEpochAbortsIf` is later changed to not assert, then this include becomes necessary?